tests/fs_utils.cpp

```diff
@@ -71,4 +72,46 @@ TEST(PathConversion, MissingFile)
 	EXPECT_FALSE(path_exists(to_native_path(nonexistent_file)));
 }
 
+constexpr char TEST_DIR[] = "tests/files/no_path";
+
+struct CreateDirTest : public testing::Test {
```

Comment:
Interesting; this is how to create a pre-requisite action before each test?

Reply:
Yup; it's called a "test fixture" (notice `TEST_F` macro is used below). Actions pre/post-test can be placed in c-tor/d-tor (modern approach) or in SetUp/TearDown methods (legacy approach). Doing it this way guarantees that cleanup happens even if test fails, so it won't cause cascading test failures due to leaving stuff behind.